src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = days * 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const bigDiscount = 50;
```
I think this would be more logical